subprojects/tooling-api/src/main/java/org/gradle/tooling/events/OperationType.java

```diff
@@ -57,6 +59,14 @@
      *
      * @since 5.1
      */
-    TRANSFORM
+    TRANSFORM,
+
+    /**
+     *  Flag for test output operation progress events.
+     *
```

Comment:
:o: It's kind of obvious but I think we should document that clients need to subscribe to `TEST` events if they want to receive `TEST_OUTPUT` events.